autofs-5.0.4 - use CLOEXEC flag setmntent include fix

From: Ian Kent <raven@themaw.net>

Fix mntent.h not included before use of setmntent_r() in automount.h.
---

 include/automount.h |    1 +
 lib/dev-ioctl-lib.c |    1 -
 lib/mounts.c        |    1 -
 3 files changed, 1 insertions(+), 2 deletions(-)


diff --git a/include/automount.h b/include/automount.h
index 1f14d5b..615e07d 100644
--- a/include/automount.h
+++ b/include/automount.h
@@ -19,6 +19,7 @@
 #include <errno.h>
 #include <unistd.h>
 #include <fcntl.h>
+#include <mntent.h>
 #include "config.h"
 #include "list.h"
 
diff --git a/lib/dev-ioctl-lib.c b/lib/dev-ioctl-lib.c
index 056a0a9..7c8c433 100644
--- a/lib/dev-ioctl-lib.c
+++ b/lib/dev-ioctl-lib.c
@@ -22,7 +22,6 @@
 #include <sys/stat.h>
 #include <sys/ioctl.h>
 #include <fcntl.h>
-#include <mntent.h>
 #include <sys/vfs.h>
 
 #include "automount.h"
diff --git a/lib/mounts.c b/lib/mounts.c
index 08ca4e3..4787bb6 100644
--- a/lib/mounts.c
+++ b/lib/mounts.c
@@ -14,7 +14,6 @@
 
 #include <stdlib.h>
 #include <string.h>
-#include <mntent.h>
 #include <limits.h>
 #include <sys/types.h>
 #include <sys/stat.h>
